src/styles/pages/_live.scss

```diff
@@ -0,0 +1,173 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-live-landing-page {
+  color: $GREY_800;
+
+  p {
+    font-size: 14px;
+    line-height: 1.5;
+    @include bp(sm) {
+      font-size: 16px;
+    }
+  }
+
+  h2 {
+    color: $GREY_900;
+  }
+
+  h3,
+  h4,
+  h5,
+  h6 {
+    color: inherit;
+  }
+}
+
+.w-live-top {
+  // We need this as the live background extends outside the normal l/r of the
+  // page. Contain it so it doesn't let users scroll.
+  overflow-x: hidden;
+}
+
+.w-event-buttons {
+  display: inline-flex;
+  flex-flow: column wrap;
+
+  .w-button--primary {
+    margin: 0 0 16px;
+  }
+
+  .w-button:not(.w-button--primary) {
+    color: inherit;
+  }
+
+  @include bp(sm) {
+    flex-direction: row;
+
+    .w-button--primary {
+      margin: 0 16px 0 0;
+    }
+  }
+}
+
+.w-event-heading {
```

Comment:
oh if the parent block isn't used you could also just make the two children into blocks of their own.

.w-event-logo
.w-event-subhead

Reply:
I think it's cleaner to leave it since these two things really only make sense in context. I've added a longer comment.